redaxo/src/core/lib/login/login.php

```diff
@@ -503,7 +503,7 @@ protected static function regenerateSessionId()
      */
     public static function startSession()
     {
-        if ('' == session_id()) {
+        if (PHP_SESSION_NONE === session_status()) {
```

Comment:
Wenn Sessions deaktiviert sind, sind wir bisher ins `if` rein gegangen. Jetzt neu gehen wir nicht mehr rein (da status nicht `PHP_SESSION_DISABLED`).
Bin mir unsicher, ob es so besser, schlechter oder egal ist.

Kann mich aber auch nicht erinnern, tatsächlich mal auf den Fall von deaktivierten Session gestoßen zu sein. Also eher ein theoretischer Fall.

Da wir vorher ins if reingegangen sind, vermute ich, dass es darin dann zu einem Fehler kam. Neu hingegen kommt dann kein Fehler, aber es läuft halt auch keine Session.
Mit Fehler war es dann vielleicht doch besser, da man auf die Problematik aufmerksam gemacht wurde.

Daher Vorschlag:
```suggestion
        if (PHP_SESSION_ACTIVE !== session_status()) {
```

So hätten wir wahrscheinlich wieder das gleiche Verhalten wie vorher.